kroxylicious-filters/kroxylicious-encryption/src/main/java/io/kroxylicious/filter/encryption/TemplateKekSelector.java

```diff
@@ -69,8 +70,8 @@ public CompletionStage<Map<String, K>> selectKek(@NonNull Set<String> topicNames
             return EnvelopeEncryptionFilter.join(collect).thenApply(list -> {
```

Comment:
Does Pair still need its generic arguments?

Reply:
No it didn't.

resolved with 355feb105a6c3f0d9150d9aea70e61af92a506e0